libs/blocks/aside/aside.css

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable selector-class-pattern */
```

Comment:
@meganthecoder this selector class pattern complains about our text content styles like `detail-M` not being kebab-case. We have quite a few of those. I am thinking we might want to update our stylelint config to leave those alone. Thoughts?

Reply:
I'd say we should not disable style-lint and fix the casing issues across the whole site styles. This can be a separate task but id rather fix the issue wholesale then pepper in style-disable comments. That'd be my take.

Reply:
I simply removed the comment and we can fix them everywhere in a separate task (being that this is not the only place where these exist)